typhon/cloudmask/aster.py

```diff
@@ -919,3 +950,15 @@ def cloudtopheight_IR(bt, cloudmask, latitude, month, method="modis"):
     bt_clear_avg = np.nanmean(bt[mask_clear])
 
     return (bt_clear_avg - bt_cloudy) / lapserate
+
+def geocentric2geodetic(latitude):
```

Comment:
There is a function for that in `typhon.geodesy`. I am not sure if it does the same, but you could check that.

Reply:
True! However, the conversion given here is from the ASTER user guide. Might be an approximation, but as it's stated there I'd like to stay consistent within the ASTER L1B data. The `typhon.geodesy` seems more complicated (might be more precise?)

Reply:
Fair enough! You may mention that in the docstring or define the function as a staticmethod of the Aster image class. But having it in the `typhon.cloudmask.aster` may already sufficient ;)